woocommerce/woocommerce-android
# Adjust analytics hub feature flag

Summary
==========
Fix issue #7699 by removing the Product stats and Visitors stats from the Analytics Hub Feature Flag, making everything accessible to the wider public, except the `See Report` button feature.

How to Test
==========
1. Simply interact with the Analytics Hub in multiple possible ways to verify if something should be fixed before releasing this new batch of features.
2. Make sure the `See Report` button is nowhere to be found when the Feature Flag is disabled.

Update release notes:
